src/ontology/edits/oeo-physical.omn

```diff
@@ -3213,6 +3221,120 @@ Class: OEO_00000449
         <http://purl.obolibrary.org/obo/RO_0000087> some OEO_00000001
     
     
+Class: OEO_00010000
+
+    Annotations: 
+        <http://purl.obolibrary.org/obo/IAO_0000115> "Ammonia is a portion of matter with the chemical formula NH3. It is a gas and can act as an air pollutant. Synonyms are trihydridonitrogen and nitrogen trihydride.",
```

Comment:
```suggestion
        <http://purl.obolibrary.org/obo/IAO_0000115> "Ammonia is a portion of matter with the chemical formula NH3. It can act as an air pollutant. Synonyms are trihydridonitrogen and nitrogen trihydride.",
```